keyvault/spring-cloud-azure-starter-keyvault-secrets/single-property-source/README.md

```diff
@@ -1,21 +1,136 @@
+# Using Key Vault Secrets With Spring Cloud Azure Key Vault Secrets Starter
 
-# Spring Cloud Azure Starter Key Vault Secrets Sample
+This sample illustrates the simplest usage of `spring-cloud-azure-starter-keyvault-secrets`. 
+To learn all features, please refer to [reference doc](https://microsoft.github.io/spring-cloud-azure/docs/current/reference/html/index.html#secret-management).
```

Comment:
Using this symbolic link, current, will make it possible to be broken in the feature. But seems like we don't have a better choice now.